modules/python/test/test_misc.py

```diff
@@ -816,6 +816,80 @@ def test_named_arguments_with_output_argument(self):
         np.testing.assert_equal(dst, src_copy)
         self.assertEqual(arguments_dump, 'lambda=25, sigma=5.5')
 
+    def test_arithm_op_with_scalar_issue_24057(self):
+        src1_mat = np.full(shape=(5,5,3), fill_value=(64,64,64), dtype='uint8')
+
+        # Basic arithm operators
+        ## add
+        dst = src1_mat + [[[192,31,30],]]  # The result of this calculation is not saturated.
```

Comment:
I don't think that we should add tests for NumPy behavior, because it has types promotion and different casting rules:
```
>>> import numpy as np
>>> a = np.array([250, 250, 250], dtype=np.uint8)
>>> a += [10, 10, 20]
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
numpy.core._exceptions._UFuncOutputCastingError: Cannot cast ufunc 'add' output from dtype('int64') to dtype('uint8') with casting rule 'same_kind'
>>> b = a + [10, 10, 20]
>>> b.dtype
dtype('int64')
```

Reply:
I agree with you. I'l remove them. Thank you !